### PR TITLE
KD: add support for machine ssh subcommand

### PR DESCRIPTION
**Pull request by ppknap**
This PR adds support for SSH remote connections via kd machine ssh subcommand.

Closes: #9889
Depends on: ~#10018, #9797, #10019, #10036~

### CODE REVIEW

**Comment by ppknap** [comment]
@rjeczalik @cihangir ptal

File: go/src/koding/klientctl/ctlcli/ctlcli.go [hunk 1]
```diff
@@ -96,6 +97,8 @@ func ExitErrAction(f ExitingErrCommand, log logging.Logger, cmdName string) func
 		exit, err := f(c, log, cmdName)
 		if err != nil {
 			log.Error("ExitErrAction encountered error. err:%s", err)
+			// Print error message to the user.
+			fmt.Fprintf(os.Stderr, "fatal: %s\n", err)
```
Comment (rjeczalik) [inline go/src/koding/klientctl/ctlcli/ctlcli.go]: `fmt.Fprintf(os.Stderr, "error executing %q command: %s\n", cmdName, err)`

File: go/src/koding/klientctl/ctlcli/ctlcli.go [hunk 2]
```diff
@@ -116,6 +119,8 @@ func FactoryAction(factory CommandFactory, log logging.Logger, cmdName string) f
 				"Command encountered error. command:%s, exit:%d, err:%s",
 				cmdName, exit, err,
 			)
+			// Print error message to the user.
+			fmt.Fprintf(os.Stderr, "fatal: %s\n", err)
```
Comment (rjeczalik) [inline go/src/koding/klientctl/ctlcli/ctlcli.go]: ditto

File: go/src/koding/klientctl/ctlcli/ctlcli.go (hunk 1 above)
Comment (rjeczalik) [inline go/src/koding/klientctl/ctlcli/ctlcli.go]: Would you also update places where we print explicitly (like credential.go) to not have duplicate entries?

**Review by cihangir — APPROVED** [review]

**Review by rjeczalik — CHANGES_REQUESTED** [review]
https://github.com/koding/koding/pull/10037#discussion_r93088909

**Comment by ppknap** [comment]
@rjeczalik done

**Review by rjeczalik — APPROVED** [review]

**Pull request merged**